### DISCUSSION TITLE
combineReducers

### DISCUSSION THREAD

**Discussion by revione** [reduxjs/redux-toolkit#3323] [Q&A]
I don't found combineReducers on the code.. is it correct?

**Comment by revione** [comment]
Now, is this like?
```
import { combineReducers } from "redux"
```

**Reply by EskiMojo14** [reply]
I'm not sure I fully understand the question, sorry.

combineReducers is a utility defined in the core "redux" package, and *re-exported* by "@reduxjs/toolkit". This means you can import it from either package.

**Reply by revione** [reply]
First, thanks for looking into this,
secondly,
I don't know why... but for a moment I got this message in the typescript verification, 
```
Module '"@reduxjs/toolkit"' has no exported member 'combineReducers'.
```

then I looked for combineReducers
in the dist/index.d.ts
and it was not there... so I started to dive into the web to find an answer to the anomaly...

but now that I crashed the machine, and came back after a while, the error message disappeared... but I still can't find where it's declared, which for now lowers the intensity of where it's impotent, although it's still important.

**Reply by phryneas** [reply]
There is `export * from 'redux'` somewhere in that file.

**Reply by revione** [reply]
Nice, I've seen it, it's in 
```
packages/toolkit/src/index.ts
```

**Reply by revione** [reply]
this also appears in the first line of index.d.ts

**Reply by revione** [reply]
![c](https://user-images.githubusercontent.com/11996529/229759660-e4320c1f-83f0-4426-af04-4b8ce8603a37.png)

**Reply by revione** [reply]
there it is again

**Reply by phryneas** [reply]
It seems that you have some local problems with your editor?
